Sources/WebSocketManager/Models/testModels/CellActivity.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  File.swift
+//  
+//
+//  Created by Maxim on 14.07.2022.
+//
+
+import Foundation
+
+public struct CellActivity {
```

Comment:
Looks like CellActivity might conform Encodable protocol, just to pass array of CellActivity to JSONEncoder